install_sedumi.m

```diff
@@ -36,6 +36,13 @@ function install_sedumi( varargin )
 
 need_rebuild = any( strcmp( varargin, '-rebuild' ) );
 no_path = any( strcmp( varargin, '-nopath' ) );
+openblas_path = '';
```

Comment:
I think it is okay to "hardcode" the default path `/usr/include/openblas` here.   This is valid for most of the Linux distros of today.  Your mechanism enables it to overwrite this value by `-obpath`, that was my assertion in #17 to not only hardcode it.

Reply:
I thought about hardcoding it, but what if it is not correct for a given distro? Could it make the installation fail where it would work without it?

Reply:
In your later code you check for `if ~isdir(openblas_path)` to be a valid directory.  Thus either the installation fails there, or later, when the inclusion of "f77blas.h" fails.  I prefer to fail here, as the user gets a useful hint, if that header is missing.

Reply:
Wait total garbage my comment.  You are right.  Remember the Windows Octave builds:  The header is already available, but not in the subdirectory "openblas".  This is will fail....

Another idea:  Leave the field empty here.  Below: if OCTAVE && `openbas_path` is empty,  just add `-I/usr/include/openblas` for safety. ;-)

Reply:
More like it :)